spec/requests/user_show_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'UserShows', type: :feature do
+  it 'response body includes correct placeholder text' do
+    visit user_path(1)
+    expect(page).to have_content('Show a particular user')
```

Comment:
Also kindly test whether the correct template is being rendered by using eg., `expect(response).to render_template(:show)`